### DISCUSSION TITLE
Warning message when using node < 14

### DISCUSSION THREAD

**Discussion by johnsonsamuel** [remix-run/remix#2543] [General]
Tried to run the following command,

```js
npx create-remix --template remix-run/indie-stack blog-tutorial
```
Got error the following message

<img width="661" alt="Screen Shot 2022-03-29 at 11 12 11 PM" src="https://user-images.githubusercontent.com/4627394/160672713-9c29795c-e475-4537-a0b9-9c505bd220c2.png">

It would be good, if there was a message to said something like, `upgrade to node14 or higher.`

**Comment by MichaelDeBoey** [comment]
Related to #2440
